bin/check-manifest-yaml.rb

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env ruby
+
+require 'pathname'
+require 'rbs'
+require 'yaml'
+
+def green(str)
+  "\e[32m#{str}\e[0m"
+end
+
+def stdlib?(name)
+  (RBS::Repository::DEFAULT_STDLIB_ROOT / name / '0').exist?
+end
+
+manifest_file = Pathname('manifest.yaml')
+if manifest_file.exist?
+  manifest = YAML.safe_load(manifest_file.read)
+
+  unless manifest && manifest.key?('dependencies') && !manifest['dependencies'].empty?
+    puts <<~END
+      #{manifest_file} exists but does not contain dependencies.
+      Please add dependencies to #{manifest_file} or remove it.
+    END
+    exit 1
+  end
+
+  manifest['dependencies'].each do |dep|
+    unless dep.key?('name')
+      puts "#{dep} does not contain the name of dependency."
+      exit 1
+    end
+
+    unless stdlib?(dep['name'])
```

Comment:
This check will be unnecessary. I'll remove it after the bug is fixed. 
https://github.com/ruby/rbs/issues/1538